### ISSUE TITLE
Too much debugging output by default, possibly causing bottleneck

### ISSUE THREAD

**Issue by IanDavey** [Xcential-Corporation/XSLTjs#1]
When I try to use XSLTjs by running 51 USC through `USLM_to_HTML.xsl` (in particular, by using Resolver's `TransformElement`, which calls Tools' `Transformer.transform`, which in turn calls XSLTjs), the console fills up with output, most of it is `# Executing: template` and `# - no match`.  Because it has to constantly send this to the console, it takes several minutes to do the transform.

tl;dr - We need a way to disable debugging output.  I've had luck with the `debug` NPM [module](https://www.npmjs.com/package/debug) in other projects, where we can set an environment variable to specify what output we want to see.

**Comment by grantcv1** [comment]
Do you need to leave the debugger at the verbose level for other debug messages?

**Comment by IanDavey** [comment]
Yes.  But while the Resolver's `debugMode` is false right now (though I sometimes set it to true to chase something down), the messages being outputted are coming from `console.debug`.

**Comment by IanDavey** [comment]
As an update - I commented out the `console.debug` statements in the `node_modules`, and while that removes debugging output, it doesn't affect performance that much, so I've edited my original post wrt. the possible bottleneck.
